showcase/gapic-showcase/src/test/resources/META-INF/native-image/resource-config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "resources":[{"pattern": ".*.json"}]
+  "resources":[{"pattern": ".*.json"}, {"pattern":  ".*.pem"}]
```

Comment:
Noticed that this is in `native-image` folder, is this change required for native image tests?

Reply:
It was not necessary!